src/server/handlers/dynamic.js

```diff
@@ -38,11 +38,20 @@ export default ({ server, config }) => {
       } else {
         log.debug(`Skipping cache: ${chalk.green(cacheKey)}`)
       }
+      // Get headers and filter by client config
+      const headers = {}
+      config.headers &&
```

Comment:
We _should_ probably check that it is an array as well, so `.map()` will be available. `Array.isArray()`?

Reply:
Good point, the user could be mischievous and pass a NaN 😾